lib/sequel/plugins/timestamps.rb

```diff
@@ -19,7 +19,7 @@ module Plugins
     #
     #   # Timestamp Artist instances, forcing an overwrite of the create
```

Comment:
The description is referring to an _Artist_ model, but on the code example it's using an _Album_ model. Just a small fix.